var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -35,6 +37,8 @@ class Nwchem(Package):
 
     variant("openmp", default=False, description="Enables OpenMP support")
     variant("mpipr", default=False, description="Enables ARMCI with progress rank")
+    variant("armcimpi", default=False, description="Enables ARMCI-MPI")
+    variant("extratce", default=False, description="Enables rarely-used TCE features")
```

Comment:
I'd default to `True` so users don't lose any functionality when upgrading spack.

Reply:
the impact on compilation time is profound.  `False` has been the default in NWChem for 20 years and no one has complained.  i dispute the backwards compatibility impact justifies the massive increase in compilation time.

Reply:
It may be better to give some explaination on what's disabled here, as we expect users of spack get all info they need by `spack info ...`, and NWChem's installation page what's non-default TCE feature is not mentioned.

Reply:
https://nwchemgit.github.io/TCE.html#overview says:
> The distributed binary executables do not contain CCSDTQ and its derivative methods, owing to their large volume. The source code includes them, so a user can reinstate them by setenv CCSDTQ yes and recompile TCE module.

>The EA/IP-EOMCCSD methodologies are available in the 6.5 NWChem release. These implementation are available for the RHF type of the reference function. To enable the compilation of the EA/IP-EOMCCSD codes one has to set the following variable before the compilation of NWChem
> export EACCSD=y
> export IPCCSD=y

I don't document the `CCSDTLR` build option because I know what the user base is.  If somebody tries to run the method, the error message tells them it's not compiled and then they'll figure out what to do about it.

Reply:
see 8acba5914fe9973d6574cc0a662b1ffbb8c460fa